frontend-student/src/components/coursePage/CourseCompletionModal.jsx

```
@@ -0,0 +1,132 @@
+          </div>
+          <h3 className='text-2xl font-bold text-gray-900 mb-2'>
+            Congratulations!
+          </h3>
+          <p className='text-gray-600 mb-6'>
+            You have successfully completed the entire course!
+          </p>
+          <div className='flex justify-center space-x-4'>
+            <button
+              onClick={onClose}
+              className='px-6 py-2 bg-blue-600 text-white rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)